go/libkb/api.go

```diff
@@ -494,7 +494,7 @@ func (a *InternalAPIEngine) fixHeaders(arg APIArg, req *http.Request) {
 		}
 	}
 	if a.G().Env.GetTorMode().UseHeaders() {
-		req.Header.Set("User-Agent", UserAgent)
+		req.Header.Set("User-Agent", UserAgent())
```

Comment:
Sorry didn't notice this earlier! We probably don't want to fudge the os for the `InternalAPIEngine` which hits our own servers.